src/acvp.c

```diff
@@ -1302,6 +1302,7 @@ ACVP_RESULT acvp_check_test_results (ACVP_CTX *ctx) {
      * in the regisration response.  Attempt to download the result
      * for each vector set.
      */
+	rv = ACVP_NO_CTX;
```

Comment:
Maybe it would make more sense to add an additional error code for missing or invalid attributes? This error could be a bit misleading

Reply:
Yes, I was thinking along the lines of the comment saying "or no valid context", but I am new to this codebase. Adding a new error makes a lot of sense. I will make this change.

Reply:
The error codes are defined around line 1300 in acvp.h :) thank you!